public_html/index.html

```diff
@@ -1,41 +1,68 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <title>Layout Pixels 2</title>
-
-    <link rel="stylesheet" href="pixels2.css">
-
+    <title>Layout Bootstrap</title>
+
+    <!-- Latest compiled and minified CSS -->
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+
+    <!-- jQuery library -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+
+    <!-- Latest compiled JavaScript -->
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+    
+    <style>
+      body {
+        /* body of document */ 
+        padding-top: 15px;
+      }
+      .img-responsive-vert {
+        /* vertical implementation of bootstrap class img-responsive */
+        width: auto;
+        max-height: 100%;
+      }
+    </style>
   </head>
   <body>
 
-  <div class="site-container">
-
-    <header class="rounded-box site-header">
-      <a class="site-header__logo" href="#"></a>
-      
-      <nav class="navbar-menu">
-	<a class="navbar-menu__item" href="#">Home</a>
-	<a class="navbar-menu__item" href="#">About</a>
-	<a class="navbar-menu__item" href="#">Pricing</a>
-	<a class="navbar-menu__item" href="#">Sign Up</a>
-	<a class="navbar-menu__item" href="#">Contact</a>
+    <div class="container-fluid">
+
+      <nav class="navbar navbar-default">
+	<div class="container-fluid">
```

Comment:
Sorry about the delays there Alistair, I'd made the commit but forgot to
push!

On 6 July 2016 at 12:43, Alistair Chisholm <notifications@github.com> wrote:

> In public_html/index.html
> <https://github.com/gordon-qcode/hello-world/pull/4#discussion_r69715355>:
>
> >    </head>
> >    <body>
> >
> > -  <div class="site-container">
> > -
> > -    <header class="rounded-box site-header">
> > -      <a class="site-header__logo" href="#"></a>
> > -
> > -      <nav class="navbar-menu">
> > -	<a class="navbar-menu__item" href="#">Home</a>
> > -	<a class="navbar-menu__item" href="#">About</a>
> > -	<a class="navbar-menu__item" href="#">Pricing</a>
> > -	<a class="navbar-menu__item" href="#">Sign Up</a>
> > -	<a class="navbar-menu__item" href="#">Contact</a>
> > +    <div class="container-fluid">
> > +
> > +      <nav class="navbar navbar-default">
> > +	<div class="container-fluid">
>
> From here: http://getbootstrap.com/css/#overview-container
>
> Note that, due to padding and more, neither container is nestable.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gordon-qcode/hello-world/pull/4/files/c142575bfc5711e30a3073ceda3a5912447a2b02#r69715355>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AS-1f3zaoEbLD-hm8heB3fB8DdlrfNjiks5qS5TfgaJpZM4JF_QC>
> .
>